.github/workflows/publish.yml

```diff
@@ -38,7 +39,11 @@ jobs:
           ref: ${{ inputs.git_ref }}
       - name: Generate publish metadata
         id: generate-publish-metadata
-        run: make ci-generate-publish-metadata
+        run: |
+          # If set, MODE will be used to set the "channel", which is the dir/namespace packages are organized into.
```

Comment:
Maybe we should just move this comment up to the `mode` argument at the head of the workflow? Nit but you could also move this env var into an `env: ` block as you see elsewhere in this file.